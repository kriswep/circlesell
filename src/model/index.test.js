/* globals test expect */
import { omit } from 'lodash';
import db from './index';

test('db model definitions should be the right ones', () => {
  expect(omit(db, ['sequelize', 'Sequelize'])).toMatchSnapshot();
});
