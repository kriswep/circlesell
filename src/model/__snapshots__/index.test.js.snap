// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`db model definitions should be the right ones 1`] = `
Object {
  "Manufacturer": [Function],
  "Product": [Function],
  "Quantity": [Function],
}
`;
